play/src/front/Phaser/Player/Player.ts

```diff
@@ -143,6 +144,15 @@ export class Player extends Character {
             x = x + 1;
         }
 
+        if (
+            activeEvents.get(UserInputEvent.MoveRight) ||
+            activeEvents.get(UserInputEvent.MoveLeft) ||
+            activeEvents.get(UserInputEvent.MoveDown) ||
+            activeEvents.get(UserInputEvent.MoveUp)
+        ) {
+            passStatusToOnlineWhenUserIsInSetableStatus();
```

Comment:
MayBe you could use `handleKeyDownEvent` of the `UserInputManager` class in the `GameScene`. All events are used with this class or handler function created to this use case.